### PR TITLE
Revised the texts in the Introduction chapter

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: examples/01_introduction/01_Hello world.md [hunk 1]
```diff
@@ -12,13 +12,11 @@ fun main() {                        // 2
...
 </div>
 
-1. Kotlin code is usually defined in packages. If you don't define one, the default package will be used.
-2. The main entry point to a Kotlin application is a function called *main* and 
-since Kotlin 1.3 it can be a function without any parameters. 
-3. `println` writes to standard output and is implicitly imported;
-also, note that semicolons are optional.
+1. Kotlin code is usually defined in packages. Package specification is optional: If you don't specify a package in a source file, its content goes to the default package.
+2. An entry point to a Kotlin application is the `main` function. In Kotlin 1.3, you can declare `main` without any parameters. The return type is not specified, which means that the function returns nothing. 
+3. `println` writes a line to the standard output. It is imported implicitly. Also, note that semicolons are optional.
```
Comment (hhariri) [inline examples/01_introduction/01_Hello world.md]: Also note that semicolons are optional.

Comment (p7nov) [reply examples/01_introduction/01_Hello world.md]: Removed the comma.

File: examples/01_introduction/05_Classes.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ fun main() {
 
 </div>  
 
-1. Declare a class named `Customer` with parameterless constructor.
```
Comment (B1ggDave) [inline examples/01_introduction/05_Classes.md]: all this list

s instead of ing

File: examples/01_introduction/06_Generics.md [hunk 3]
```diff
@@ -46,16 +46,16 @@ fun main() {
 
 </div>
 
-1. Defines a generic class `MutableStack<E>` where `E` is called the _generic type parameter_. At use-site, it is assigned to a concrete type such as `Int` by declaring a `MutableStack<Int>`.
+1. Defining a generic class `MutableStack<E>` where `E` is called the _generic type parameter_. At use-site, it is assigned to a specific type such as `Int` by declaring a `MutableStack<Int>`.
```
Comment (B1ggDave) [inline examples/01_introduction/06_Generics.md]: passive

File: examples/01_introduction/06_Generics.md [hunk 4]
```diff
@@ -46,16 +46,16 @@ fun main() {
...
 3. You can also use `E` as a return type.
 
 Note that the implementation makes heavy use of Kotlin's shorthand syntax for functions that can be defined in a single expression.
 
 
-### Generic functions
+### Generic Functions
 
-You can also [generify functions](https://kotlinlang.org/docs/reference/generics.html#generic-functions) if their logic is independent of the concrete type. For instance, you can create a utility function to create mutable stacks more easily:
+You can also [generify functions](https://kotlinlang.org/docs/reference/generics.html#generic-functions) if their logic is independent of the specific type. For instance, you can write a utility function to create mutable stacks easier:
```
Comment (B1ggDave) [inline examples/01_introduction/06_Generics.md]: if their logic is independent of a specific type

Comment (B1ggDave) [reply examples/01_introduction/06_Generics.md]: For instance, you can easily write a utility function to create mutable stacks:

File: examples/01_introduction/07_Inheritance.md [hunk 5]
```diff
@@ -83,7 +80,6 @@ fun main() {
 </div>
 
 
-1. Observe that `name` is neither a `var` nor `val` because it is a
-    constructor argument for `Asiatic`, whose value is passed to the `name`
-    attribute of `Lion` (it does not override it).
+1. `name` in the `Asiatic` declaration is neither a `var` nor `val`: it's a constructor argument, whose value is passed to the `name` property of the superclass `Lion`. 
+2. Creating an `Asiatic` instance with the name `Rufo` invokes the `Lion` constructor with arguments `Rufo` and `India`.
```
Comment (B1ggDave) [inline examples/01_introduction/07_Inheritance.md]: passive. boom!

**Pull request merged**